core/src/main/scala/com/databricks/labs/remorph/parsers/tsql/TSqlRelationBuilder.scala

```diff
@@ -1,18 +1,74 @@
 package com.databricks.labs.remorph.parsers.tsql
 
-import com.databricks.labs.remorph.parsers.{IncompleteParser, intermediate => ir}
 import com.databricks.labs.remorph.parsers.tsql.TSqlParser._
+import com.databricks.labs.remorph.parsers.{IncompleteParser, intermediate => ir}
 
-import scala.collection.JavaConverters._
+import scala.collection.JavaConverters.asScalaBufferConverter
 
 class TSqlRelationBuilder extends TSqlParserBaseVisitor[ir.Relation] with IncompleteParser[ir.Relation] {
 
   protected override def wrapUnresolvedInput(unparsedInput: String): ir.Relation = ir.UnresolvedRelation(unparsedInput)
 
-  override def visitTableSourceItem(ctx: TableSourceItemContext): ir.Relation = {
-    val table = ctx.fullTableName().getText
-    // [TODO]: Handle Table Alias ctx.tableAlias().getText
-    ir.NamedTable(table, Map.empty, is_streaming = false)
+  override def visitSelectStatementStandalone(ctx: TSqlParser.SelectStatementStandaloneContext): ir.Relation = {
+
+    // TODO: Process ctx.WithExpression
+    ctx.selectStatement().accept(this)
+    // val tableSources = Option(rawExpression.tableSources()).fold[ir.Relation](ir.NoTable())
```

Comment:
Is this commented code part of the above TODO? Otherwise we may not want to keep it.

Reply:
It was. But looking again, I don't think it needs to be retained here.